parquet-column/src/main/java/org/apache/parquet/column/statistics/BinaryStatistics.java

```diff
@@ -190,4 +209,12 @@ public void setMinMax(Binary min, Binary max) {
   public BinaryStatistics copy() {
     return new BinaryStatistics(this);
   }
+
+  public void setGeometryStatistics(GeospatialStatistics geospatialStatistics) {
```

Comment:
We should name it `setGeospatialStatistics` because we already have `getGeospatialStatistics`.